### PR TITLE
clean up unecessary config code

### PR DESCRIPTION
**Pull request by udaydenduluri33**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: api/local.ts [hunk 1]
```diff
@@ -1,16 +1,15 @@
 /**
  * Common to both server.ts and local.ts files
  */
-import * as propertiesVolume from '@hmcts/properties-volume'
-import * as config from 'config'
 import { app } from './application'
-import {getConfigValue, getIDamSecret, initialiseSecrets} from './configuration'
+import {getConfigValue, initialiseSecrets} from './configuration'
```
Comment (paddybasi) [inline api/local.ts]: don't need the `initialiseSecrets` function anymore

File: api/configuration/index.ts [hunk 2]
```diff
@@ -34,14 +34,35 @@ export const hasConfigValue = reference => {
...
-    return secretsConfig['secrets']['rpx']['mc-idam-client-secret']
-  } else {
-    console.log(ERROR_IDAM_SECRET_NOT_FOUND)
-    return ''
-  }
-}
+/**
+ * Allows us to integrate the Azure key-vault flex volume, so that we are able to access Node configuration values.
+ */
+propertiesVolume.addTo(config, { mountPoint: '/Volumes/mnt/secrets/'})
```
Comment (paddybasi) [inline api/configuration/index.ts]: please remove the `mountPoint` otherwise this will fail in AKS

File: api/local.ts (hunk 1 above)
Comment (udaydenduluri33) [reply api/local.ts]: Done

File: api/configuration/index.ts (hunk 2 above)
Comment (udaydenduluri33) [reply api/configuration/index.ts]: Done

**Pull request merged**